### PR TITLE
First release

### PR DESCRIPTION
**Pull request by LucasZibell**
## Summary

Firestore Service supports INITIALIZE, CREATE, GET, DELETE, POST and PATCH. With success test for each of them.

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.env
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+package.json
```
Comment (mcallegari10) [inline .npmignore]: This file should be published

File: package.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "firestore-service",
+  "version": "1.0.0",
```
Comment (mcallegari10) [inline package.json]: Is this the final version that will be published?

File: test.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  const response = await firestoreService.GET(path);
+  expect(response).toEqual({
+    data: { age: 33, firstName: 'Mike', id: 'J8NR45UzDffyMY0wBNoa', lastName: 'Poe' },
+    request: 'GET',
+    status: 200,
+    statusText: 'OK'
+  });
+});
+
+test('Get All Users', async () => {
```
Comment (mcallegari10) [inline test.js]: Be more self explainable with test cases

File: test.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  const path = 'tests/get/users';
+  const response = await firestoreService.GET(path);
+  expect(response).toEqual({
```
Comment (mcallegari10) [inline test.js]: Move this object to a mock file

File: test.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    data: { age: 33, firstName: 'Mike', id: 'J8NR45UzDffyMY0wBNoa', lastName: 'Poe' },
+    request: 'GET',
+    status: 200,
+    statusText: 'OK'
+  });
+});
+
+test('Get All Users', async () => {
+  const path = 'tests/get/users';
```
Comment (mcallegari10) [inline test.js]: I think this in an unnecessary `const`

File: test.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    statusText: 'OK'
+  });
+});
+
+test('Create Doc', async () => {
+  const path = 'tests/create/animals';
+  const body = { name: 'Bob the test Bear', specimen: 'Bear' };
+  const response = await firestoreService.CREATE(path, body);
+  const newUser = await firestoreService.GET(`tests/create/animals?id=${response.data}`);
+  expect(response).toEqual({
```
Comment (mcallegari10) [inline test.js]: Same for this object, move to a mock files

File: test.js [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+        lastName: 'June'
+      }
+    ],
+    request: 'GET',
+    status: 200,
+    statusText: 'OK'
+  });
+});
+
+test('Create Doc', async () => {
```
Comment (mcallegari10) [inline test.js]: Same, explain better the test case please :smile:

File: package.json [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  "description": "",
```
Comment (mcallegari10) [inline package.json]: Add a short description of the package

File: package.json [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.7.0",
+    "dotenv": "^6.0.0",
+    "eslint-config-airbnb": "^17.0.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-flowtype": "^2.49.3",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-prettier": "^2.6.1",
+    "eslint-plugin-react": "^7.10.0",
```
Comment (mcallegari10) [inline package.json]: Remember to check the dependencies, there are some of them you don't really need

**Comment by mcallegari10** [comment]
Remember to add a `README.md` file with documentation. At least the basics for this first release :smile: :tada:

**Comment by SKOLZ** [comment]
btw do what @mcallegari10 said about the readme please, this is very usefull!.

**Pull request merged**